MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,13 @@
+# Go-Auth0 v1 Migration Guide
+
+With the v1 release of Go-Auth0, we've
```

Comment:
```suggestion
With the v1 release of Go-Auth0, we've
```

SUSPENSE!!! We did what? 😟

Reply:
Removed this for now and will write a preamble in future when polishing the migration guide